src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+
+let totalPopulation = 0;
+let totalCountries = 0;
+
+for (const countryPopulation of document.querySelectorAll('.population')) {
+  totalPopulation += +countryPopulation.innerText.replaceAll(',', '');
```

Comment:
The use of `replaceAll` might not be supported in all JavaScript environments, especially older ones. Consider using `replace` with a regular expression: `countryPopulation.innerText.replace(/,/g, '')`.